util/argo/normalizers/corev1_known_types.go

```diff
@@ -0,0 +1,622 @@
+// Code generated by github.com/argoproj/argo-cd/hack/known_types. DO NOT EDIT.
```

Comment:
I know this is generated code, but I think we should have simple unit tests for it as well. For one, we can see whether the generated code is up-to-date with the list of known types, and also missing tests would decrease this package's coverage unnecessarily.

I think test could be as simple as the following pseudo-code:
```
knownTypes := readKnownTypesSourceFile();
for kt := range knownTypes {
  _, ok := knownTypes[kt]
  assert.True(t, ok)
}
```
What do you think?

Reply:
good idea. Done